packages/dd-trace/test/appsec/iast/analyzers/unvalidated-redirect-analyzer.express.plugin.spec.js

```diff
@@ -28,54 +26,75 @@ describe('Unvalidated Redirect vulnerability', () => {
     prepareTestServerForIastInExpress('in express', version,
       (testThatRequestHasVulnerability, testThatRequestHasNoVulnerability) => {
         testThatRequestHasVulnerability((req, res) => {
-          const store = storage('legacy').getStore()
-          const iastCtx = iastContextFunctions.getIastContext(store)
-          const location = newTaintedString(iastCtx, 'https://app.com?id=tron', 'param', 'Request')
+          const location = req.query.location// newTaintedString(iastCtx, 'https://app.com?id=tron', 'param', 'Request')
           redirectFunctions.insecureWithResHeaderMethod('location', location, res)
         }, UNVALIDATED_REDIRECT, {
           occurrences: 1,
           location: {
             path: redirectFunctionsFilename,
             line: 4
           }
+        }, null, (done, config) => {
+          axios.get(`http://localhost:${config.port}/?location=https://app.com?id=tron`).catch(done)
```

Comment:
I think it is worth creating an instance of `axios` with the `baseUrl` to avoid repeating it throughout the test suite.

Reply:
I've modified it creating an axios instance in a method to reuse it.